packages/grid/test/scroll-to-index.test.js

```diff
@@ -213,8 +213,8 @@ describe('scroll to index', () => {
 
     it('should not reassign the first item on scrollToIndex', () => {
       const newExpectedSize = grid.size + 1;
-      grid.size = newExpectedSize;
       data.push({ index: 11 });
+      grid.size = newExpectedSize;
```

Comment:
The size change triggered an immediate data provider request while the data array was only updated on the following line.